europeana-entity-harvester/index.html

```diff
@@ -66,7 +68,9 @@
           if (uriMatch) return uriMatch[1];
 
           const pageMatch = entityUrl.match(new RegExp('^https?://[^/]+/entity(/(person|topic)/[0-9]+)'));
+
           if (pageMatch) {
+            entityType = pageMatch[2];
```

Comment:
Harvesting by URI fails to set the entity type, causing previews to fail.

I suggest not setting this variable here at all, and instead, at https://github.com/europeana/contentful-extensions/pull/6/files#diff-00ad045f38c1ee484ebd1553e9201a92R176-R178, doing:
```js
          if (extension.entry.fields.type) {
            extension.entry.fields.type.setValue(entityTypeFromUri(response.id)); // entity type
          }
```